### ISSUE TITLE
Need information on function preprocess_image

### ISSUE THREAD

**Issue by rgsl888** [fizyr/keras-retinanet#428]
Hi, 
        I had a question, how [preprocess_image](https://github.com/fizyr/keras-retinanet/blob/master/keras_retinanet/utils/image.py) helps and what it does?

**Comment by hgaiser** [comment]
It subtracts the mean RGB values of the images in ImageNet from the current image. The goal is to transform the pixel values such that their mean value is equal to zero. At one point in time people found out that doing this helps classifiers in their accuracy. I don't think that's still the case if you use BatchNorm, but ResNet uses it so we have to use it to if we use ResNet.

**Comment by rgsl888** [comment]
Thank you for the information

**Comment by hgaiser** [comment]
You're in the right direction, but Lambda layers always gives me issues (specifically with saving / loading models). My advice : make a small custom layer for this (look at ClipBoxes for instance).

**Comment by rgsl888** [comment]
Thank you for the information, I have created custom layers.
